index.html

```diff
@@ -32,7 +32,149 @@ <h2 class="first-name software-dev">I'm a software developer</h2>
+   <h3 class="multi-text-color">Gain+Glory</h3>
+   <ul class="language display-flex">
+   <li class="language-background">Ruby on rails</li>
+   <li class="language-background">css</li>
+   <li class="language-background" >JavaScript</li>
+   <li class="language-background">html</li>    
+ </ul>  
+ <button class="see-project">See Project</button>
+</div>
+</div>
+
+<div class="grid-project">
+  <div class="multi-post">
+   <h3 class="multi-text-color">Multi-Post Stories</h3>
+   <h3 class="multi-text-color">Gain+Glory</h3>
+   <ul class="language display-flex">
+   <li class="language-background">Ruby on rails</li>
+   <li class="language-background">css</li>
+   <li class="language-background" >JavaScript</li>
+   <li class="language-background">html</li>    
+ </ul>  
+ <button class="see-project">See Project</button>
+</div>
+</div>
+
+<div class="grid-project">
+  <div class="multi-post">
+   <h3 class="multi-text-color">Multi-Post Stories</h3>
+   <h3 class="multi-text-color">Gain+Glory</h3>
+   <ul class="language display-flex">
+   <li class="language-background">Ruby on rails</li>
+   <li class="language-background">css</li>
+   <li class="language-background" >JavaScript</li>
+   <li class="language-background">html</li>    
+ </ul>  
+ <button class="see-project">See Project</button>
+</div>
+</div>
+
+<div class="grid-project">
+  <div class="multi-post">
+   <h3 class="multi-text-color">Multi-Post Stories</h3>
+   <h3 class="multi-text-color">Gain+Glory</h3>
+   <ul class="language display-flex">
+   <li class="language-background">Ruby on rails</li>
+   <li class="language-background">css</li>
+   <li class="language-background" >JavaScript</li>
+   <li class="language-background">html</li>    
+ </ul>  
+ <button class="see-project">See Project</button>
+</div>
+</div>
+
+<div class="grid-project">
+   <div class="multi-post">
+    <h3 class="multi-text-color">Multi-Post Stories</h3>
+    <h3 class="multi-text-color">Gain+Glory</h3>
+    <ul class="language display-flex">
+    <li class="language-background">Ruby on rails</li>
+    <li class="language-background">css</li>
+    <li class="language-background" >JavaScript</li>
+    <li class="language-background">html</li>    
+  </ul>  
+  <button class="see-project">See Project</button>
+</div>
+</div>
+</section>
+
+<!-- About me Session -->
+<section class="about-container">
+    <img class="float-right" src="images/Mask_Group_floate_right.png" alt="elipse-floating-right">  
+      <h2 class="about-me">About Me</h2><br><br><br>
+         <p class="about-message">I can help you build a product, feature or website 
+         Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don't hestiate to contact me.</p>
+         <div><img class="float-left" src="images/Group_float_left.png" alt="floating-elispe-left">
+  <button class="see-project get-resume">Get My Resume</button>  
```

Comment:
### OPTIONAL
- [ ]  Kindly add  the button styling for : Active, Hover and Pressed (see screenshot below)

![Screenshot_54](https://user-images.githubusercontent.com/69638013/165087220-ca342432-ff0f-4660-aa11-fd3205002661.png)